src/Appium.Net/Appium/Android/AndroidDriver.cs

```diff
@@ -213,7 +214,21 @@ public ConnectionType ConnectionType
 
         protected override RemoteWebElementFactory CreateElementFactory() => new AndroidElementFactory(this);
 
-        #region locking
+        #region Device Performance Data
+
+        public IList<object> GetSupportPerformanceData(string packageName, string dataType,
```

Comment:
What the `Support` prefix means here?

Reply:
Ah, this is supposed to stated supported, but even that term is confusing, in order for someone to know what is supported they'd have to call GetPerformanceDataTypes.. thinking of making the datatype parameter an enum instead of a string

Reply:
I would say `GetPerformanceDataTypes` would be enough

Reply:
done